tests/integration/test_application.py

```diff
@@ -81,6 +81,7 @@ def app(create_app):
 
 For more information about a command, run 'testcraft help <command>'.
 For a summary of all commands, run 'testcraft help --all'.
+For more information about testcraft, check out: www.craft-app.com/docs/3.14159
```

Comment:
Probably best not to point this to another site of unknown provenance. I've replaced with [RFC 2606](https://datatracker.ietf.org/doc/html/rfc2606) domains.
```suggestion
For more information about testcraft, check out: testcraft.example/docs/3.14159
```